salt/metalk8s/kubernetes/cni/calico/deployed.sls

```diff
@@ -666,8 +3712,6 @@ spec:
               value: "info"
             - name: FELIX_HEALTHENABLED
               value: "true"
-            - name: FELIX_USAGEREPORTINGENABLED
```

Comment:
We need to keep this one @TeddyAndrieux ...

Reply:
Ok I will add it again

Reply:
#2827